feat: stop storing algo category (#24)
Substra/orchestrator: CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]
**WARNING**: Some migrations in this version are destructive once applied you will not be able to restore algo categories.

### Changed

> - Algo.category: You should not rely on categories anymore, all algo categories will be returned as UNKNOWN

Reviewer: ```suggestion
- (BREAKING) Algo.category: do not rely on categories anymore, all algo categories will be returned as UNKNOWN
```
